ParameterDataApi.yaml

```diff
@@ -0,0 +1,246 @@
+---
+openapi: 3.0.1
+info:
+  title: FRAIHMWORK Parameter Data API
+  description: 'FRAIHMWORK allows for parameter time-series data to be provided from the client. This API will versioned according to SemVer 2.0 rules (as described here: https://semver.org/)'
+  contact:
+    email: matt.synborski@resilienx.com
+  version: 0.0.0
+externalDocs:
+  description: Find out more about Swagger
+  url: http://swagger.io
+
+servers:
+- url: https://<site>.fraihmwork.resilienx.com/api/parameter/v0
+
+tags:
+- name: Parameter
+  description: Parameter data model manipulation endpoints
+
+paths:
+  /parameter:
+    post:
+      tags:
+        - Parameter
+      summary: Add new time series parameter data to FRAIHMWORK model
+      description: Add new time series parameter data to FRAIHMWORK model
+      operationId: registerParameterData
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/TimeSeriesDataDefinition'
+      responses:
+        201:
+          description: POSTed new time series parameter data
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+          
+        400:
+          description: Bad Request (invalid submission). 
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+          
+        401:
+          description: Not authenticated
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        403:
+          description: Forbidden
+          content: 
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        406:
+          description: Not acceptable - only responds in application/json format
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        500:
+          description: Server error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+      security:
+      - ParameterDataDefinitionsAuth:
+        - fraihmwork.parameter.data.write
+
+components:
+  schemas:
+    Response:
+      type: object
+      description: A default uuid / message combination to convey simple success / error messages to the client
+      properties:
+        itemId:
+          type: string
+          format: uuid
+        message:
+          type: string
+          description: The success message provided back to the client
+      required:
+        - message
+        
+    ColumnDefinition:
+      type: object
+      description: The column information for the data.      
+      properties:
+        col0:
```

Comment:
You're trying to have things both ways (array for some references of columns, and a property based one here). You should be consistent across both the header definition and the data

Reply:
Also, any reason for limiting to just one column? This is a deviation from eagle.io, wasn't sure if it was intentional